misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -312,3 +329,7 @@ class MiskWebModule @JvmOverloads constructor(
     val miskCallerType = object : TypeLiteral<MiskCaller?>() {}
   }
 }
+
+@BindingAnnotation
+@Target(AnnotationTarget.FIELD, AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
+annotation class ReadinessRefreshQueue
```

Comment:
this can be internal

Reply:
Should it be? It could be useful for apps to override the service with their own implementation, and I don't see any reason for the framework to prohibit this.

Reply:
The values on the `ReadinessRefreshQueue` is exposed in the `readiness_refresh_interval_ms` config. I'm not sure what would be value of exposing this.

Reply:
We discussed this out of band, Ive marked it as internal